templates/organization/packages.html.twig

```diff
@@ -28,10 +28,10 @@
                     <td>
                         {% if package.name %}
                             <strong>{{ package.name }}</strong><br />
-                            <small class="font-italic" style="font-size: 0.7em">{{ package.url }}</small><br />
+                            <small class="font-italic" style="font-size: 0.7em"><a href="{{ package.url }}" target="_blank">{{ package.url }}</a></small><br />
                             <small class="font-italic" style="font-size: 0.7em" title="{{ package.lastSyncAt | date('Y-m-d H:i:s') }}">synced: {{ package.lastSyncAt | time_diff }}</small>
                         {% elseif package.lastSyncError is null %}
-                            Synchronizing from {{ package.url }}
+                            Synchronizing from <a href="{{ package.url }}" target="_blank">{{ package.url }}</a>
```

Comment:
It will not always make sense (for example git url), but I think it can be useful.